ui/src/pages/Home/Profile/profile.scss

```diff
@@ -11,9 +11,27 @@
     padding: 10px;
     height: auto;
     width: auto\9; /* ie8 */
+    &__loading {
+      display: none;
+    }
   }
   &__text {
     text-align: justify;
     padding-right: 10px;
   }
+  &__skeleton {
+    width: 100%;
+    height: 200px;
+    background-color: rgb(247, 199, 208);
```

Comment:
cand tell if there is a skeleton. this color too similar to background

Reply:
i mean i guess that was the point of it? Should i make it a darker or lighter color? or should i just have it be the loading text?

Reply:
I like how this looks `rgb(255, 176, 191)`. going lights the colors don't have much of a difference. going darker makes the color look 'dirty'. 

the skeleton looks a bit wonky fix the right edge. try fixing the right edge. 
![image](https://github.com/user-attachments/assets/3b765b58-5e4e-493e-8801-8ad0f023e2b3)

Reply:
@jmorales8 was this implemented?

Reply:
yes